samza-sql-shell/src/main/java/org/apache/samza/sql/client/cli/CliEnvironment.java

```diff
@@ -173,11 +187,21 @@ public SqlExecutor getExecutor() {
     return executor;
   }
 
+  public List<CommandHandler> getCommandHandlers() { return commandHandlers; }
+
   private void createShellExecutor(String executorClassName) throws ExecutorException {
+    executor = (SqlExecutor) createInstance(executorClassName);
```

Comment:
I just noticed the original code didn't handle the `ClassCastException`. Shell would crash if a user flirts with the configuration file, setting `executor` or `commandhandler` to some random class. Could you help fix it here?

Reply:
done